### ISSUE TITLE
Integration startet nicht 

### ISSUE THREAD

**Issue by Fc19852008** [sfstar/hass-victron#55]
Hallo ich habe es über die HACS repo instaliert. Ich bekomme aber immer die Meldung: 
Logger: homeassistant.setup
Source: setup.py:251
First occurred: 19:43:04 (1 occurrences)
Last logged: 19:43:04

Setup failed for custom integration victron: No setup or config entry setup function defined.

Was mache ich falsch?

**Comment by Fc19852008** [comment]
2023-03-06 20:26:19.755 WARNING (SyncWorker_1) [homeassistant.loader] We found a custom integration victron which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2023-03-06 20:26:19.755 WARNING (SyncWorker_1) [homeassistant.loader] We found a custom integration hacs which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2023-03-06 20:26:28.947 ERROR (MainThread) [homeassistant.setup] Setup failed for custom integration victron: No setup or config entry setup function defined.
